lib/net.c

```diff
@@ -1116,6 +1116,14 @@ start_iface(struct gatekeeper_if *iface, unsigned int num_attempts_link_get)
 	uint8_t i;
 	uint8_t num_succ_ports = 0;
```

Comment:
Style: moving the initialization of `num_succ_ports` to immediately before the `for` loop below would enable the compiler to warning any introduction of a `goto` above label `destroy_init` that `num_succ_ports` was not initialized. This is good because that would be a bug.